9c697d609329ca10c7067a418bba1022/dictionary/x_snc_plant_mgmt_plants.xml

```diff
@@ -1,28 +1,76 @@
 <?xml version="1.0" encoding="UTF-8"?><database>
     <element audit="true" db_object_id="036cf1a49329ca10c7067a418bba10da" label="Plants" max_length="40" name="x_snc_plant_mgmt_plants" type="collection">
-        <element attributes="array=denormalized,no_sort=true,slushbucket_ref_no_expand=true" label="Fertilising schedule" max_length="1024" name="u_fertilising_schedule" type="glide_list"/>
-        <element label="Location" max_length="32" name="location" reference="cmn_location" type="reference"/>
-        <element choice="1" label="Status" max_length="40" name="status" type="choice">
+        <element choice="1" label="Tissue Culture" max_length="40" name="tissue_culture" type="choice">
             <choice>
-                <element inactive_on_update="false" label="Poor" sequence="1" value="poor"/>
-                <element inactive_on_update="false" label="Dead" sequence="3" value="dead"/>
-                <element inactive_on_update="false" label="Good" sequence="0" value="good"/>
-                <element inactive_on_update="false" label="Sick" sequence="2" value="sick"/>
+                <element inactive_on_update="false" label="IC0001" sequence="0" value="ic0001"/>
+                <element inactive_on_update="false" label="IC0003" sequence="2" value="ic0003"/>
+                <element inactive_on_update="false" label="IC0002" sequence="1" value="ic0002"/>
```

Comment:
@404paige I just saw this label updated in a different PR, and the fork for this one must be older than that, hence having some changes here. Should we remove these or accept? Not sure about the relevance.

Reply:
Let me know, because I closed one older PR just to change the branch because I reviewed the video and assumed that it wouldn't have been accepted if I didn't change my branch from main to something else for organizational issues. There is one other change on here as well from the other or else it could not have been committed.

Reply:
No it's OK, nothing you did wrong, just a conflict that will probably come back in other PRS too :)